doc/specs/preview-monaco.md

```diff
@@ -0,0 +1,64 @@
+# **Preview pane - Adding previews with monaco**
+
+<img align="right" src="../images/overview/PT_small.png" />
+
+- **What is it:** Implementing [#1527](https://github.com/microsoft/PowerToys/issues/1527)
+- **Authors:** Aaron Junker ([@aaron-junker](https://github.com/aaron-junker)), Clint Rutkas ([@crutkas](https://github.com/crutkas))
+- **Spec Status:** Waiting for review
+
+# 1 Overview
+
+Creating a new preview handler for all Developer files. For this we use [Microsoft Monaco editor](https://github.com/microsoft/monaco-editor). 
+
+## 1.1 Technical implementation
+
+A WebView2 window implements shows HTML file with Monaco integrated.
+
+### 1.1.1 New dependencies
+
+The installer needs to install [WebView2](https://developer.microsoft.com/en-us/microsoft-edge/webview2/)
+
+### 1.1.2 Why WebView2
+
+Experiments with a POC showed that only WebView2 is displaying Monaco the right way.
```

Comment:
I'd list what you did try and explain the results here.  Assume no one has context and this is to drive understanding,